### DISCUSSION TITLE
Can't open QRPGLESRC file with '£' in the name of the programm

### DISCUSSION THREAD

**Discussion by jmansiat** [codefori/vscode-ibmi#2368] [IBM i]
Hello from France,
i 'm a new user of Vscode for i for few days now.
I have a problem to open QRPGLESRC files whose names contain an '£'
I receive the message : The editor could not be opened because the file was not found.
The file exist on the IBM i and can be opened by using PDM.
I can't rename the program because used in my ERP.
Is anyone have the same problem and find an issue ?
Thanks a lot
Jerome

**Comment by sebjulliand** [comment]
Hi @jmansiat ,
you should try the latest pre-release, available on the marketplace. It should fix your issue.
When displaying the extension details, click on this button.
![image](https://github.com/user-attachments/assets/058b2e49-690d-484b-b9f7-7b985494940f)

Then restart VS Code and reconnect. Your members with variant characters should open correctly.

**Reply by jmansiat** [reply]
Hello @sebjulliand and thanks for your answer.
I have install the pre-release version for Code for IBM i and switch to.
Restatrt vscode and re-connect to IBM i
But i have some problems now : 
- user library list is shown as empty but have the good liblist in
- object browser show 2 library but can't develop them to find QRPGLESRC and the program member. Receive the error : invalid time value
-
![image](https://github.com/user-attachments/assets/1b1a61da-7a62-46a9-81ec-d59fcc34eca6)

**Reply by worksofliam** [reply]
@jmansiat seems like your issue may be similar to this: https://github.com/codefori/vscode-ibmi/issues/2360#issuecomment-2489833786
